articles/sentinel/whats-new.md

```diff
@@ -27,13 +27,30 @@ Noted features are currently in PREVIEW. The [Azure Preview Supplemental Terms](
 
 ## March 2021
 
+- [New detections for Azure Firewall](#new-detections-for-Azure-Firewall)
 - [Automation rules and incident-triggered playbooks](#automation-rules-and-incident-triggered-playbooks) (including all-new playbook documentation)
 - [New alert enrichments: enhanced entity mapping and custom details](#new-alert-enrichments-enhanced-entity-mapping-and-custom-details)
 - [Print your Azure Sentinel workbooks or save as PDF](#print-your-azure-sentinel-workbooks-or-save-as-pdf)
 - [Incident filters and sort preferences now saved in your session (Public preview)](#incident-filters-and-sort-preferences-now-saved-in-your-session-public-preview)
 - [Microsoft 365 Defender incident integration (Public preview)](#microsoft-365-defender-incident-integration-public-preview)
 - [New Microsoft service connectors using Azure Policy](#new-microsoft-service-connectors-using-azure-policy)
- 
+
+### New detections for Azure Firewall
+Several out-of-the-box detections for Azure Firewall have been added to the Analytics in Azure Sentinel. These new detections allow security teams to get alerts if machines on the internal network attempt to query/connect to domain names or IP addresses on the internet that are associated with known IOCs, as defined in the detection rule query. Here are the new detections: <p>
+| No.| Sentinel Analytic Rule Name| Sentinel Repo Link|
+| --| --|--|
```

Comment:
Suggest turning this table into a bulleted list.

Also - do we need the links to github? Maybe a screenshot and a single link to github for the root folder?

Reply:
Each rule actually its own link in the Sentinel github repo. It's hard for users to find if we provide a link to a folder. That's why I include the links to each of the rule on github. The blog post already has the screenshot of the rule templates, and if adding a screenshot, this PR will need to be moved to the private repo per Andrea :).